app/util.py

```diff
@@ -119,3 +126,27 @@ def project_user_permissions(form):
         permissions[user] = user_permissions
 
     return permissions
+
+
+# Padding for encryption
+# The lambdas are from StackOverflow, I have no idea how they work
+def pad(e):
+    # Block size
+    BS = 16
+    return (lambda s: s + (BS - len(s) % BS) * chr(BS - len(s) % BS))(e)
```

Comment:
Take a string (e becomes s inside the lambda). Take the block size minus the length of the string mod the block size (how many chars you need to pad the string out to the block size) and multiply by the ascii charcter representation of the number of chars you need to pad (i.e. if you need to ad 10 chars, return chr(10), which is '\n'). Then add those char strings to the end of the original string, thereby padding it out to the block size with ascii char strings.